src/util/std_code.h

```diff
@@ -282,7 +282,7 @@ class code_assumet:public codet
   inline code_assumet():codet(ID_assume)
   {
     // will change to resize(1) in the future
-    operands().reserve(1);
+    reserve_operands(1);
```

Comment:
Is the comment above still relevant/correct? Is `reserve_operands` what `resize` ended up being called?